blackbird_python/blackbird/tests/test_program.py

```diff
@@ -189,19 +189,27 @@ def test_serialize_operation_multiple_kwargs(self):
         assert res == expected
 
     def test_serialize_operation_args_kwargs(self):
-        """Test serialization of an operation with args and kwargs raises an error"""
+        """Test serialization of an operation with args and kwargs"""
         bb = BlackbirdProgram(name="prog", version=1.0)
         bb._operations.append(
             {
-                "op": "Dgate",
+                "op": "MeasureHomodyne",
                 "modes": [0],
-                "args": [0.54],
-                "kwargs": {"alpha": 0.43 - 0.543j, "phi": 0.54},
+                "args": [0.54, -1],
+                "kwargs": {"select": 5.43, "phi": 0.54},
             }
         )
 
-        with pytest.raises(ValueError, match="contain both positional and keyword arguments"):
-            bb.serialize()
+        res = bb.serialize()
+        expected = dedent(
+            """\
+            name prog
+            version 1.0
+
+            MeasureHomodyne(0.54, -1, select=5.43, phi=0.54) | 0
```

Comment:
Does `MeasureHomodyne` actually take this many arguments? Or is it just used because it is an operation which can take both args and kwargs?

Reply:
Yeah, MeasureHomodyne is currently the only operation which can take both. I added more than one positional/keyword argument just to make sure this works.